faros_event.sh

```diff
@@ -347,6 +351,11 @@ function parseBuildUri() {
 
 function parseCommitUri() {
     parseUri "${commit_uri:-$FAROS_COMMIT}" "vcs_source" "vcs_org" "vcs_repo" "commit_sha" "source://org/repo/commit"
+
+    if !((no_lowercase_vcs)); then
```

Comment:
does this also check for `no_lowercase_vcs` not being set at all?

Reply:
no_lowercase_vcs defaults to 0 so if not set then it will lowercase these fields.